plugin/src/main/java/com/denizenscript/denizen/events/server/ListPingScriptEvent.java

```diff
@@ -118,6 +119,7 @@ public ObjectTag getContext(String name) {
             case "max_players" -> new ElementTag(event.getMaxPlayers());
             case "num_players" -> new ElementTag(event.getNumPlayers());
             case "address" -> new ElementTag(event.getAddress().toString());
+            case "hostname" -> new ElementTag(event.getHostname());

```

Comment:
Doesn't look like this method existed on 1.17? at a glance it looks like it got added on 1.19, but should make sure and add a version check

Reply:
And nitpick, but should probably be plain text (e.g. `new ElementTag(str, true)`)